docs/development_suite/overview-dev-suite.md

```diff
@@ -4,28 +4,47 @@ title: Console Overview
 sidebar_order: 0
 ---
-Welcome to the Mia-Platform Console Overview page!
+This page will give you a brief overview of **Mia-Platform Console**, explain **why it's important**, highlight the **advantages of adopting this internal platform solution** and provide a **brief description of its architecture**. 
 
-![Mia-Platform Console Overview](img/Mia-Platform-console-overview.png)
+## Product overview 
 
-In an omnichannel and evolutionary market, successful enterprises are the ones with the capabilities to operate and innovate like tech companies:
+In an evolving, omnichannel market, successful enterprises must operate and innovate like tech companies by:
 
-* **Speed**: Being able to build and launch new digital products with great speed;
-* **Evolvability**: Continuously looking at the ongoing market changes as opportunities;
-* **Scalability**: Being flexible and ready to answer rapidly to demand expansions or contractions;
-* **Openness**: Being able to integrate with tools, partner channels, platforms and ecosystems.
+* Building and launching digital products **quickly**; for example, reducing the time-to-market for a new app from months to weeks.
+* Viewing market changes as **opportunities**, such as rapidly adopting new technologies.
+* Being **flexible** to rapidly respond to demand fluctuations.
+* **Integrating** with various tools, partners, and ecosystems.
 
-Modern companies build their digital products, data and logics through a Digital Platform.  
-The tool to reach this goal is the **Internal Developer Portal** which will help you to industrialize and govern cloud-native development and operations. It allows to:
+Modern companies achieve this through an Internal Developer Platform (IDP). The tool to accomplish this is **Mia-Platform Console**, an Internal Developer Platform builder which aids in industrializing, managing and simplifying cloud-native development and operations.
 
-* Design and Expose APIs;
-* Develop and Orchestrate microservices;
-* Manage Real-time Data Streams and Event-driven architectures;
-* Govern and monitor runtime on K8s.
+![Mia-Platform Console Overview](img/Mia-Platform-infographic-overview.png)
 
-Mia-Platform Console allows you to govern all projects in one place, industrialize & automate DevOps, avoid organizational bottlenecks, self-serve developers, solve cloud complexity and enhance accountability & reliability.
+## Why Mia-Platform Console is important
 
-**Hierarchical Organization of Mia-Platform Console:**
+Mia-Platform Console manages the entire software lifecycle seamlessly and unify your IT department under one integrated platform. Simplify cloud-native development while maintaining complete governance and control over your processes.
+
+* **Clear Governance**: Establish streamlined, standardized pathways for software delivery and integrate governance rules directly into your development workflows, ensuring consistency, compliance, and efficiency throughout the entire development process.
```

Comment:
```suggestion
* **Clear Governance**: Establish streamlined, standardized pathways for software delivery and integrate governance rules directly into your development workflows, ensuring consistency, compliance, and efficiency throughout the entire development process. For instance, this can involve tracking the evolution of projects, identifying key characteristics of each project and centrally visualizing resources, all of which help maintain transparency and facilitate effective management.
```